sormas-ui/src/main/java/de/symeda/sormas/ui/exposure/ExposuresField.java

```diff
@@ -123,6 +147,14 @@ private void addGeneratedColumns(Table table) {
 			return exposureTypeLabel;
 		});
 
+		table.addGeneratedColumn(COLUMN_PROBABLE_INFECTION_ENVIRONMENT, (Table.ColumnGenerator) (source, itemId, columnId) -> {
```

Comment:
In line with the implementation details in the issue, I'd prefer to have an additional icon in the exposure type column instead of a new column (because we have serious issues with space in the exposures table already)

Reply:
Ah, sorry. I misunderstood this. So an exposure in a high risk area that is the probable infection environment shall have two icons? I guess those two attributes are likely to come together.

Reply:
I just implemented the two icon appraoch. I think it looks okay.

Reply:
Exactly